bulbs/liveblog/templates/liveblog/bulbs_liveblog.html

```diff
@@ -0,0 +1,49 @@
+{% load ads core betty liveblog_tags %}
+<div class="liveblog-cover">
+  {% cropped content.thumbnail '3x1' 500 %}
+</div>
+<div class="liveblog">
+  <div>LIVE COVERAGE</div>
+  <h1 class="liveblog-title">{{ content.title | safe }}</h1>
+
+
+  {% if content.pinned_content %}
+    {% with pinned_content=content.pinned_content %}
+    <div class="liveblog-pinned-article">
+      <a href="#">
+        <div class="liveblog-pinned-image">
+          {% cropped pinned_content.thumbnail "1x1" 100 %}
+        </div>
+
+        <div class="liveblog-pinned-meta">
+          <div class="liveblog-pinned-title">
+            <i class="fa fa-thumb-tack" aria-hidden="true"></i>
+            {{ pinned_content.title | safe }}
+          </div>
+
+          {% if content.subhead %}
+            <div class="liveblog-pinned-excerpt">
+              {{ content.subhead | removetags:"p br" | safe }}
+            </div>
+          {% endif %}
+
+          <div class="liveblog-published-date"></div>
+        </div>
+      </a>
+    </div>
+    {% endwith %}
+  {% endif %}
+</div>
+
+{% with entries=content.entries.all %}
+  {% liveblog_entries_partial %}
+{% endwith %}
```

Comment:
@mparent61 the partial endpoint should only have to render this templatetag with an array of entries in context.